### ISSUE TITLE
Bug: ERROR cannot write configuration to file: open /updater/data/config.json: permission denied

### ISSUE THREAD

**Issue by DANIELorlov07** [qdm12/ddns-updater#740]
**TLDR**: *Updater won't really start*

1. Is this urgent: Yes
2. DNS provider(s) you use: Strato.de
3. Program version: 'latest' 7th of June 2024
4. What are you using to run the container: docker/portainer
5. Extra information (optional)

Logs:

========================================
========================================
============= ddns-updater =============
========================================
=========== Made with ❤️ by ============
======= https://github.com/qdm12 =======
========================================
========================================
Running version latest built on 2024-05-23T13:33:03.304Z (commit 8e09cd6)
🔧 Need help? https://github.com/qdm12/ddns-updater/discussions/new
🐛 Bug? https://github.com/qdm12/ddns-updater/issues/new
✨ New feature? https://github.com/qdm12/ddns-updater/issues/new
☕ Discussion? https://github.com/qdm12/ddns-updater/discussions/new
💻 Email? quentin.mcgaw@gmail.com
💰 Help me? https://www.paypal.me/qmcgaw https://github.com/sponsors/qdm12
2024-06-07T23:52:41Z INFO Settings summary:
├── HTTP client
|   └── Timeout: 10s
├── Update
|   ├── Period: 5m0s
|   └── Cooldown: 5m0s
├── Public IP fetching
|   ├── HTTP enabled: yes
|   ├── HTTP IP providers
|   |   └── all
|   ├── HTTP IPv4 providers
|   |   └── all
|   ├── HTTP IPv6 providers
|   |   └── all
|   ├── DNS enabled: yes
|   ├── DNS timeout: 3s
|   └── DNS over TLS providers
|       └── all
├── Resolver: use Go default resolver
├── Server
|   ├── Listening address: :8000
|   └── Root URL: /
├── Health
|   └── Server listening address: 127.0.0.1:9999
├── Paths
|   └── Data directory: /updater/data
├── Backup: disabled
└── Logger
    ├── Level: info
    └── Caller: hidden
2024-06-07T23:52:41Z INFO reading JSON config from environment variable CONFIG
2024-06-07T23:52:41Z ERROR cannot write configuration to file: open /updater/data/config.json: permission denied
2024-06-07T23:52:41Z INFO Shutdown successful

Configuration file (**remove your credentials!**):

```
{
    "settings": [
        {
          "provider": "strato",
          "domain": "'domain'.de",
          "host": "'subdomain','subdomain','subdomain'",
          "password": "'password'",
          "ip_version": "ipv4",
          "ipv6_suffix": "",
          "provider_ip": true
        }
    ]
}

```

Host OS: Proxmox LXC Container/Ubuntu Server 22.04 Template/Docker

**Comment by DANIELorlov07** [comment]
You can forget it! Got it to work. Just did everythin new.

Sadly I don't know how to delete this post here xD

**Comment by qdm12** [comment]
Sorry for the delay, and no worry, I'm glad you self-fixed it 😉
